.github/workflows/docker.yml

```diff
@@ -0,0 +1,35 @@
+name: GraphScope CI on Local
+
+on:
+  schedule:
+    # Trigger the workflow at 03:00 every day
+    - cron:  '00 03 * * *'
+
+jobs:
+  build-images:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+    steps:
+    - uses: actions/checkout@v2.3.2
+
+    - name: Prepare Image
+      run: |
+        sudo docker build -t registry.cn-hongkong.aliyuncs.com/graphscope/graphscope:${{ github.sha }} \
+                          --network=host \
+                          -f .k8s/graphscope.Dockerfile .
+        sudo docker build -t registry.cn-hongkong.aliyuncs.com/graphscope/maxgraph_standalone_manager:${{ github.sha }} \
+                          --network=host \
+                          -f .k8s/manager.Dockerfile .
+        sudo docker build -t registry.cn-hongkong.aliyuncs.com/graphscope/graphscope-store:${{ github.sha }} \
+                          --network=host \
+                          -f .k8s/graphscope-store.Dockerfile .
+
```

Comment:
in addition to building the docker image, 
could you add a brief testing, e.g., run a simple session test with this image??
Since it is only a ci build, and wont upload to the registry, you may use a simpler tag to make it more clear.

Reply:
- add a simple create session test for the ci
- since we use prepare_env.sh to launch a kind cluster, the tag still need to be consistent with release image